torch/fx/_symbolic_trace.py

```diff
@@ -478,6 +478,12 @@ def _module_getattr(self, attr, attr_val, parameter_proxy_cache):
                         val_proxy = self.create_proxy('get_attr', n, (), {}, **kwargs)  # type: ignore[arg-type]
                         parameter_proxy_cache[n] = val_proxy
                     return parameter_proxy_cache[n]
+        if isinstance(attr_val, torch.Tensor):
```

Comment:
maybe refactor this with the almost identical logic above. Potentially you want the param_shapes_constant and ParameterProxy logic to apply to buffers as well.